.env.example

```diff
@@ -24,4 +24,8 @@ TEST_EMBER_ENDPOINT=localhost:50051
 # Some protocol tests require WETH.
 # Anvil generates some fake ETH that must be wrapped before use.
 WETH_ADDRESS=0x82af49447d8a07e3bd95bd0d56f35241523fbab1
-TEST_ANVIL_FORK_BLOCK_NUMBER=319869636
\ No newline at end of file
+TEST_ANVIL_FORK_BLOCK_NUMBER=22343500
+
+# Additional test startup requirements
+ETH_RPC_URL=https://eth-mainnet.public.blastapi.io
```

Comment:
The RPC is supposed to be any EVM compatible rpc, not necessarily mainnet. We use arbitrum, hence the block number above.